sdks/python/apache_beam/transforms/external.py

```diff
@@ -709,11 +729,21 @@ def __enter__(self):
       if self._extra_args is None:
         self._extra_args = self._default_args()
       # Consider memoizing these servers (with some timeout).
+      logging.info(
+          'Starting a JAR-based expansion service from JAR %s and '
+          'with classpath: %s',
```

Comment:
Probably only log the "with classpath ..." part if classpath is not empty.

Reply:
done